pipelines/services/syncstatus.go

```diff
@@ -26,25 +26,24 @@ func NewSyncStatusService(client *jfroghttpclient.JfrogHttpClient) *SyncStatusSe
 
 // GetSyncPipelineResourceStatus fetches pipeline sync status
 func (ss *SyncStatusService) GetSyncPipelineResourceStatus(repoName, branch string) ([]PipelineSyncStatus, error) {
-	// fetch resource ID
-	resID, isMultiBranch, resourceErr := GetPipelineResourceID(ss.client, ss.GetUrl(), repoName, ss.getHttpDetails())
+	// Fetch resource ID
+	res, resourceErr := GetPipelineResource(ss.client, ss.GetUrl(), repoName, ss.getHttpDetails())
 	if resourceErr != nil {
 		log.Error("Unable to fetch resourceID for: ", repoName)
```

Comment:
Let's concat this error message to the message of the error returned by this function, instead of adding this log.Error line. We prefer not to add log lines throughout the code, but instead, have the errors returned include all the relevant information. The error message should be logged at the highest upstream level as possible.